hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/ECBlockOutputStream.java

```diff
@@ -128,8 +128,13 @@ ContainerCommandResponseProto> executePutBlock(boolean close,
       List<ChunkInfo> currentChunks = getContainerBlockData().getChunksList();
       List<ChunkInfo> checksumBlockDataChunks = checksumBlockData.getChunks();
 
-      Preconditions.checkArgument(
-          currentChunks.size() == checksumBlockDataChunks.size());
+      if (currentChunks.size() != checksumBlockDataChunks.size()) {
+        throw new IllegalArgumentException("The chunk list has " +
+            currentChunks.size() +
+            " entries, but the checksum chunks has " +
+            checksumBlockDataChunks.size() +
+            " entries. They should be equal in size.");
+      }
```

Comment:
Why not just add the message in Preconditions.checkArgument itself,
```
      Preconditions.checkArgument(
          currentChunks.size() == checksumBlockDataChunks.size(),
          "The chunk list has " + currentChunks.size()
              + " entries, but the checksum chunks has "
              + checksumBlockDataChunks.size()
              + " entries. They should be equal in size.");

```

Reply:
I didn't realize it took an extra argument. I will change it.